File: go2zfl/fixtures/x1_crossfunc.go
package main
import ("net/http"; "os/exec")
func runCmd(c string) { exec.Command("sh", "-c", c) }   // param -> shell (summary)
func h5(w http.ResponseWriter, r *http.Request) {
	cmd := r.FormValue("cmd")
	runCmd(cmd)                                  // EXPECT: REFUTED (cross-func summary)
	runCmd("safe")                               // EXPECT: clean
}

File: js2zfl/fixtures/x1_crossfunc.js
const cp = require('child_process');
function runCmd(c) { cp.exec(c); }            // param -> shell (summary)
module.exports = (req, res) => {
  const cmd = req.query.cmd;
  runCmd(cmd);                                // EXPECT: REFUTED (cross-func summary)
  runCmd('safe');                             // EXPECT: clean
};
